### ISSUE TITLE
Warning: Detected code that calls hass.http.register_static_path

### ISSUE THREAD

**Issue by tjorim** [kongo09/philips-airpurifier-coap#138]
Warning since HA 2024.7, it does blocking I/O in the event loop:

```
Logger: homeassistant.helpers.frame
Source: helpers/frame.py:184
First occurred: 14:12:26 (9 occurrences)
Last logged: 14:13:56

Detected that custom integration 'philips_airpurifier_coap' calls hass.http.register_static_path which is deprecated because it does blocking I/O in the event loop, instead call `await hass.http.async_register_static_paths([StaticPathConfig("/philips_airpurifier_coap/main.js", "/config/custom_components/philips_airpurifier_coap/main.js", True)])`; This function will be removed in 2025.7 at custom_components/philips_airpurifier_coap/__init__.py, line 64: hass.http.register_static_path(LOADER_URL, hass.config.path(LOADER_PATH), True), please create a bug report at https://github.com/kongo09/philips-airpurifier-coap/issues
Detected that custom integration 'philips_airpurifier_coap' calls hass.http.register_static_path which is deprecated because it does blocking I/O in the event loop, instead call `await hass.http.async_register_static_paths([StaticPathConfig("/philips_airpurifier_coap/icons/pap", "/config/custom_components/philips_airpurifier_coap/icons/pap", True)])`; This function will be removed in 2025.7 at custom_components/philips_airpurifier_coap/__init__.py, line 89: hass.http.register_static_path(ICONS_URL + "/" + iset, iconpath, True), please create a bug report at https://github.com/kongo09/philips-airpurifier-coap/issues
```

**Comment by kongo09** [comment]
Thanks

**Comment by kongo09** [comment]
fixed in v0.18.8
